src/p11-kit.mk

```diff
@@ -0,0 +1,25 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := p11-kit
+$(PKG)_VERSION  := 0.23.2
+$(PKG)_CHECKSUM := ba726ea8303c97467a33fca50ee79b7b35212964be808ecf9b145e9042fdfaf0
+$(PKG)_SUBDIR   := p11-kit-$($(PKG)_VERSION)
+$(PKG)_FILE     := p11-kit-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://p11-glue.freedesktop.org/releases/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc libtasn1 libffi
+
+define $(PKG)_UPDATE
+    echo 'TODO: write update script for $(PKG).' >&2;
+    echo $($(PKG)_VERSION)
+endef
+
+define $(PKG)_BUILD
+    cd '$(1)' && ./configure \
+        $(MXE_CONFIGURE_OPTS) \
+#        --disable-rpath \
```

Comment:
This line and 3 following lines seem to be commented out. Can you remove or uncomment them?